qt/app.py

```diff
@@ -1286,12 +1286,27 @@ def updateTimeLine(self, refreshSnapshotsList=True):
                 item = self.timeLine.addSnapshot(sid)
             self.timeLine.checkSelection()
 
+    def validate_on_take_snapshot(self):
+        missing = snapshots.has_missing_includes(self.config.include())
+        if missing:
+            msg_missing = '\n'.join(missing)
+            msg = _('The following directories are missing: {dirs} Do you want to proceed?'.format(
+                dirs=f'\n{msg_missing}\n\n'))
```

Comment:
The `format()` need to be outside of the translatable string.

    # Problematic code
    _('foo {val}'.format(val='bar'))

    # Good code
    _('foo {val}').format(val='bar')

Don't worry, this translation stuff is complex.

```suggestion
            msg = _('The following directories are missing: {dirs} Do you want to proceed?').format(
                dirs=f'\n{msg_missing}\n\n')
```